### PR TITLE
LPS-120326 LRDOCS-8396 Headless intro article

### PR DESCRIPTION
**Pull request by ethanbustad**
https://issues.liferay.com/browse/LPS-120326
https://issues.liferay.com/browse/LRDOCS-8396

### CODE REVIEW

File: docs/dxp/7.x/en/headless-delivery/introduction-to-headless.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+There are three different approaches available for clients to connect to Liferay DXP via web API:
+* [__Headless REST APIs__](#headless-rest-apis), conforming to the OpenAPI specification
+* The [__GraphQL API__](#graphql-api) conforming to the GraphQL specification
+* [__Plain Web/REST Services__](#plain-webrest-services), generated by Service Builder
+
+### Headless REST APIs
+
+Liferay DXP's headless APIs allow [RESTful](https://www.w3.org/TR/2004/NOTE-ws-arch-20040211/#relwwwrest) interaction with Liferay DXP resources. These APIs follow the [OpenAPI specification](https://swagger.io/docs/specification/about/), which defines a standard for REST interfaces, allowing for more straightforward implementation and consumption.
+
+You can find detailed information on these APIs by accessing the definitions on a running Liferay DXP instance at `[server][:port]/o/api` or exploring their documentation on [SwaggerHub](https://app.swaggerhub.com/apis/liferayinc/).
```
Comment (javierdearcos) [inline docs/dxp/7.x/en/headless-delivery/introduction-to-headless.md:16]: The endpoint [server][:port]/o/[api-name]/[api-version]/openapi.[yaml or json] contains the OpenAPI specification of the given API of the running instance. This is very helpful as it contains the information of the exact version deployed.

SwaggerHub is usually not updated to latest version and there are common mistakes of people consulting the SwaggerHub documentation and checking that the version they are using does not match with the specifiacion, so adding a warning might be helpful

File: docs/dxp/7.x/en/headless-delivery/introduction-to-headless.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+### GraphQL API
+
+The GraphQL API is accessed using [GraphQL](https://graphql.org/), a query language that supports interactions similar to the headless REST APIs, but with slightly more flexibility.
+
```
Comment (javierdearcos) [inline docs/dxp/7.x/en/headless-delivery/introduction-to-headless.md:21]: Maybe a more explicit mention to the GraphQL endpoint (`[server][:port]/o/graphql`) at the beginning is helpful

**Review by javierdearcos — APPROVED** [review]
SME review completed. 

Just added some comments in case you consider them helpful

**Comment by ethanbustad** [comment]
@javierdearcos thanks for the suggestions! I've added a commit to incorporate those changes. Feel free to look it over, but the changes were pretty simple so I'm comfortable moving on to Rich's review (FYI, @sez11a).

**Comment by sez11a** [comment]
Superseded by https://github.com/jrhoun/liferay-learn/pull/732

**Pull request closed without merging**